### PR TITLE
Support comfyui sd3 speedup

### PR DESCRIPTION
**Pull request by ccssu**
## 安装

<div>
    <a href="https://www.kaggle.com/code/ccsufengwen/sd3-comfyui/notebook"> SD3 Speedup Quick Start</a>
    <a href="https://www.kaggle.com/code/ccsufengwen/sd3-comfyui/notebook">Open In Kaggle</a>
</div> 

## Run WorkFlow

- 28 step image 1024x1024  mode: max-optimize:max-autotune:low-precision

| Accelerator           | Baseline (non-optimized) | OneDiff (optimized) | Percentage improvement |
| --------------------- | ------------------------ | ------------------- | ---------------------- |
| NVIDIA A800-SXM4-80GB | ~4.03 sec                | ~2.93 sec           | ~27.29 %               |

```
动态 shape
 print("Test run with multiple resolutions...")
        sizes = [1024, 512, 768, 256]
        for h in sizes:
            for w in sizes:
```

workflow (17)

### CODE REVIEW

**Comment by strint** [comment]
需要补充一个 readme

**Comment by lixiang007666** [comment]
- https://oneflow-static.oss-cn-beijing.aliyuncs.com/sd3/clip_g.safetensors
- https://oneflow-static.oss-cn-beijing.aliyuncs.com/sd3/clip_l.safetensors
- https://oneflow-static.oss-cn-beijing.aliyuncs.com/sd3/sd3_medium.safetensors
- https://oneflow-static.oss-cn-beijing.aliyuncs.com/sd3/t5xxl_fp8_e4m3fn.safetensors

@ccssu 这些是工作流里可能用到的模型。

File: onediff_comfy_nodes/extras_nodes/nodes_nexfort_booster.py [hunk 1]
```diff
@@ -7,7 +7,8 @@
     {
         "jit:disable-runtime-fusion:low-precision": "This compiles super quickly, but the performance might not be optimized very noticeably.",
         "jit:benchmark:low-precision:freezing:cudagraphs": "This compiles the model very quickly, but the performance might be not as good as `TorchInductor` optimized models.",
-        "max-autotune:low-precision": "This will deliver a good performance and adapt quickly to shape changes.",
+        "max-optimize:max-autotune:low-precision": "This will deliver a good performance and adapt quickly to shape changes.",
```
Comment (strint) [inline onediff_comfy_nodes/extras_nodes/nodes_nexfort_booster.py:10]: 可以在 readme 里面介绍推荐使用这个模式

File: onediff_comfy_nodes/sd3_demo/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+## Accelerate SD3 by using onediff
+huggingface: https://huggingface.co/stabilityai/stable-diffusion-3-medium
```
Comment (strint) [inline onediff_comfy_nodes/sd3_demo/README.md:2]: 可以加个 setup 的说明：https://github.com/siliconflow/onediff/tree/main/onediff_diffusers_extensions/examples/sd3#set-up-sd3

说清楚下 nexfort 安装、comfyui、模型 id

File: onediff_comfy_nodes/sd3_demo/README.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
+## Accelerate SD3 by using onediff
+huggingface: https://huggingface.co/stabilityai/stable-diffusion-3-medium 
+
+
+### Feature
+- ✅ Multiple resolutions
+
+### Performance
```
Comment (strint) [inline onediff_comfy_nodes/sd3_demo/README.md:8]: 这个可以参考： https://github.com/siliconflow/onediff/tree/main/onediff_diffusers_extensions/examples/sd3#performance-comparation

**Pull request merged**